.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -36,13 +36,28 @@ Push access to the upstream repository is required in order to publish the new t
 
 - create release commits on a dedicated branch and tag it (the commits and tag will be signed with the GPG signing key you configured):
   - [ ] `git checkout -b release-${RELEASE_VER}`
-  - [ ] `cargo release` (and confirm the version when prompted)
+  - [ ] `cargo release -- ${RELEASE_VER}` (and confirm the version when prompted)
 
 - open and merge a PR for this release:
   - [ ] `git push ${UPSTREAM_REMOTE} release-${RELEASE_VER}`
   - [ ] open a web browser and create a PR for the branch above
   - [ ] make sure the resulting PR contains exactly two commits
-  - [ ] in the PR body, write a short changelog with relevant changes since last release
+  - [ ] in the PR body, write a short changelog with relevant changes since last release, using the following template:
```

Comment:
I think it's a good template but I wouldn't codify it here. Looking at previous entries for a guideline is often enough, and a bit of variation to adapt to individual cases is not a bad thing.

Reply:
Fair, removed!